kontrakter/dom/dom/arbeidsversjon/dom.schema.json

```diff
@@ -0,0 +1,1057 @@
+      "description": "Aktører fra domstolen",
+      "properties": {
+        "dommer": {
+          "$ref": "#/definitions/ansattPerson",
+          "description": "Dommer på saken"
+        },
+        "saksbehandler": {
+          "$ref": "#/definitions/ansattPerson",
+          "description": "Saksbehandler på saken"
+        }
+      },
+      "type": "object",
+      "additionalProperties": false
+    },
+    "siktede": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/privatPerson"
+      }
+    },
+    "dokumenter": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/dokument"
+      },
+      "minItems": 1
+    },
+    "saksInformasjon": {
+      "$ref": "#/definitions/saksInformasjon"
+    }
+  },
+  "required": [
+    "forsendelse",
+    "aktoerer",
+    "siktede",
+    "saksInformasjon",
+    "dokumenter"
+  ],
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "saksInformasjon": {
+      "properties": {
+        "saksnummer": {
+          "type": "string",
+          "description": "Domstolen sitt interne saksnummer"
+        },
+        "straffesaksnummer": {
+          "type": "string",
+          "description": "Politiet sitt interne saksnummer"
+        },
+        "rettsmoeter": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/rettsmoete"
+          },
+          "minItems": 0,
+          "description": "Dersom rettsmøte ikke avholdes, f.eks. ved kontorforrenting, skal dette bli en top array",
+          "$comment": "Trenger påtale disse? Og kan vi filtrere på møtetyper - f.eks. kun hovedforhandling"
+        },
+        "avgjoerelse": {
+          "$ref": "#/definitions/avgjoerelse"
+        }
+      },
+      "required": ["saksnummer", "straffesaksnummer", "avgjoerelse"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "avgjoerelse": {
+      "properties": {
+        "avgjoerelseId": {
+          "type": "string",
+          "description": "Domstolen sin nøkkel til avgjørelsen. Kan bli brukt i forbindelse med anke/forlengelse. AvgjørelseID beholdes ved korigeringer"
+        },
+        "avgjoerelseIdForrigeInstans": {
+          "type": "string",
+          "description": "Domstolen sin nøkkel til avgjørelsen i forrige instans. Settes kun dersom dette er en kjennelse i en ankesak"
+        },
+        "avsagtDato": {
+          "$ref": "#/definitions/dato"
+        },
+        "kravid": {
+          "type": "string",
+          "description": "Kravid i nåværende instans. Kan være null dersom saken er opprettet manuelt"
+        },
+        "opprinneligKravid": {
+          "type": "string",
+          "description": "Kraviden i første instans"
+        },
+        "avgjoerelseType": {
+          "$ref": "#/definitions/avgjoerelseType"
+        },
+        "resultat": {
+          "$ref": "#/definitions/resultat"
+        },
+        "reaksjoner": {
```

Comment:
Er tanken at domfelte er dømt for alle forhold i "resultat", og at "reaksjoner" er straffeutmålingen for summen avv alle disse forholdene vedkommende er dømt for?

Ble det noen form for strukturert gjenbruk på "lovbud"-elementet her nå, eller er dette bare tekststrenger p.t.?